torchdata/datapipes/iter/load/README.md

```diff
@@ -0,0 +1,76 @@
+# S3 IO Datapipe Documentation
+
+## Installation
+
+Torchdata S3 IO datapipes depends on [aws-sdk-cpp](https://github.com/aws/aws-sdk-cpp). The following is just a
+recommended way to installing aws-sdk-cpp, please refer to official documentation for detailed instructions.
+
+```bash
+git clone --recurse-submodules https://github.com/aws/aws-sdk-cpp
+cd aws-sdk-cpp/
+mkdir sdk-build
+cd sdk-build
+cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_ONLY="s3;transfer"
+make
+make install # may need sudo
+```
```

Comment:
We need to add a few comments here to tell users to build static library on Windows.

Reply:
Addressed in the latest commit.